pallets/data-preservers/src/benchmarks.rs

```diff
@@ -55,5 +76,209 @@ mod benchmarks {
         assert_eq!(Pallet::<T>::boot_nodes(para_id), boot_nodes);
     }
 
-    impl_benchmark_test_suite!(Pallet, crate::mock::new_test_ext(), crate::mock::Test);
+    #[benchmark]
+    fn create_profile(x: Linear<1, 200>, y: Linear<1, 10>) {
```

Comment:
According to the weights, none of the benchmarks depend on x or y, that's great!

Reply:
In the runtime weigts it seems to use x and y but with very small multipliers. Should we simplify and remove those parameters?

Reply:
I would leave it, or maybe use one param which is "profile size", but it's fine